jarviscli/Jarvis.py

```diff
@@ -139,16 +140,24 @@ def parse_input(self, data):
         else:
             # if it doesn't have a fixed response, look if the data corresponds
             # to an action
-            output = self.find_action(
+            action_plugin, action_index, output = self.find_action(
                 data, self._plugin_manager.get_plugins().keys())
+            if hasattr(action_plugin, '_feature'):
+                plugin_features = \
+                  self._parse_plugin_features(action_plugin.feature())
+                if plugin_features['case_sensitive']:
+                    output = action_plugin.get_name() + " " + \
+                             " ".join(words[action_index + 1:])
         return output
```

Comment:
You are only adding ``action_plugin.get_name()`` to plugins with feature: 'case_sensitive'. This basically breaks Jarvis for all other commands.